src/SpectatorView.Unity/Assets/SpatialAlignment.ASA/Scripts/AzureSpatialAnchors/CustomSpatialAnchorManager.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.Azure.SpatialAnchors.Unity
+{
+    public static class CustomSpatialAnchorManagerExtensions
+    {
+        /// <summary>
+        /// Extension method that allows adding a SpatialAnchorManager with a custon SpatialAnchorConfig to a GameObject
+        /// </summary>
+        /// <param name="gameObject"></param>
```

Comment:
Nit: Don't have empty params or return statements (just delete them if you're not gonna add details). The generated docs look the same generally (i.e. using docfx or something else) so these don't add much value aside from padding the line count.